wiki/Beatmap_Editor/fr.md

```diff
@@ -8,14 +8,16 @@ L'**éditeur de beatmap** est l'interface graphique intégrée au client et est
 
 L'éditeur de beatmap est séparé en 4 sections qui peuvent être vues dans le coin supérieur gauche de l'éditeur :
 
-- [Compose](#compose)
-- [Design](#design)
-- [Timing](#timing)
-- [Song setup](#song-setup)
+- [Éditeur de Beatmap](#éditeur-de-beatmap)
```

Comment:
this is not a table of contents, remove the first/last items

Reply:
Same thing for `Banchobot`, it will be corrected